components/eventing-controller/Makefile

```diff
@@ -131,3 +131,6 @@ $(eval $(call buildpack-mount,manifests))
 
 resolve_clean:
 	rm -rf vendor
+
+lint-thoroughly:
```

Comment:
nit: I think `lint-strict` might also be a good name and is a bit shorter. Feel free to choose your name or my version. Just an idea :)